build.cake

```diff
@@ -97,10 +97,10 @@ Task("Test")
 Task("PublishCalamariProjects")
    .IsDependentOn("Build")
     .Does(() => {
-        var projects = GetFiles("./source/**/Calamari.*.csproj");
+        var projects = GetFiles("./source/**/Calamari.csproj");
```

Comment:
If this is going to be the template repo, we will need a new function to zip the Calamari test project

Reply:
Actually I think we should have a template repository in Github.

The 🍰  files should all be consistent though

Reply:
This is actually something I really like about this approach, and the adjustments that have been made in these cake files versus the ones we've got in the extension repos. The ps1 files are same in both cases but the cake files here are much cleaner, no mention of the repo specifics which means would could auto flow the same file to all repos and it'll work nicely.